extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -254,7 +255,11 @@ public void initialize() {
     private synchronized void onUpdate() {
         if (bridge != null) {
             if (pollingJob == null || pollingJob.isCancelled()) {
-                pollingJob = scheduler.scheduleAtFixedRate(pollingRunnable, 1, POLLING_FREQUENCY, TimeUnit.SECONDS);
+            	int pollingInterval = DEFAULT_POLLING_INTERVAL;
+            	try {
+            		pollingInterval = ((BigDecimal) getConfig().get(POLLING_INTERVAL)).intValue();
```

Comment:
The ThingImpl only guarantees that the configuration object (the map) is not null.
I assume that someone could edit the configuration (e.g. REST) and could set an empty one (the usage of default value is only guarantee if someone is using the XML to create the thing).